app/controllers/api/auth_controller.rb

```diff
@@ -5,7 +5,7 @@ class AuthController < BaseController
     def show
       requester_type = fetch_and_validate_requester_type
       token_service = Environment.user_token_service
-      auth_token = token_service.generate_token(User.current_user.userid, requester_type)
```

Comment:
aside, there is `User.current_userid` for future ref
you are deleting this reference, so not relevant here